ReleaseNotes-2.9

```diff
@@ -79,6 +79,10 @@ Wanted changes:
 
  * password hashing algorithm was changed to bcrypt (blowfish)
 
+ * Notification of events to the backend won't be sent anymore, we have replaced this with the RabbitMQ message bus.
+   Instead of the backend sending notifications through the plugins after receiving those notifications,
+   now the frontend will send the messages through the bus directly.
```

Comment:
The backend notification plugin system is not used anymore. The RabbitMQ plugin is replaced with a RabbitMQ message bus implementation in the frontend, you can find details about this in the our admin manual. The Hermes plugin is dropped without replacement as it was only used for notifications which the OBS is doing on it's own since quite some time.

Reply:
But the hermes and RabbitMQ plugins in the backend aren't dropped already, aren't they?

Reply:
You mean the code removed? No. But by removing `NotifyBackendJob` OBS will not make use of them anymore so effectively they are dropped. Remember, this file is for users.